libs/wingc/src/jsify/snapshots/access_methods_and_properties_on_collections.snap

```diff
@@ -29,6 +29,8 @@ module.exports = function({ $_y_at_0__, $x_length }) {
       {((cond) => {if (!cond) throw new Error("assertion failed: x.length == 3")})((((a,b) => { try { return require('assert').deepStrictEqual(a,b) === undefined; } catch { return false; } })($x_length,3)))};
       {((cond) => {if (!cond) throw new Error("assertion failed: y.at(0) == \"hello\"")})((((a,b) => { try { return require('assert').deepStrictEqual(a,b) === undefined; } catch { return false; } })($_y_at_0__,"hello")))};
     }
+    async $inflight_init() {
+    }
```

Comment:
I'm considering removing all these empty `inflight_init`s, but the nicest way of doing so will be to correctly reflect they don't exist in the AST because they really don't exist.
I'll try to add that to the PR..

Reply:
Eventually added support for removing empty `inflight_init`s without modifying the AST because that just made the code too complex. Instead I just avoid executing it if it's not defined in the emitted JS object.